### DISCUSSION TITLE
New window(pop up) is not visible in Cypress

### DISCUSSION THREAD

**Discussion by OlegBuch** [cypress-io/cypress#17968] [Questions and Help] [answered]
1. We have an integration with PayPal on the web-site
2. When we press on PayPal button and a new window should appear - a new pop up in the browser. Nothing happens in Cypress.
Can you please point us to what we can do to make this new window visible. This new window has another url not the one we are using during our tests.

**Comment by tit** [comment, marked as answer]
Hi!
In short, there is no way.
See https://stackoverflow.com/questions/59667218/how-to-handle-elements-on-paypal-popup-with-cypress-io

**Comment by RaafaPimentel** [comment]
I am also having the same problem.

**Reply by OlegBuch** [reply]
Well what we did is just used another framework. We used protractor.
